GitCommands/GitCommands.csproj

```diff
@@ -5,6 +5,12 @@
     <!-- To be removed when NRT annotations are complete -->
     <Nullable>annotations</Nullable>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DefineConstants>$(DefineConstants);TRACE_REVISIONREADER</DefineConstants>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <DefineConstants>$(DefineConstants);TRACE_REVISIONREADER</DefineConstants>
```

Comment:
Will maybe want to remove this later, keep for some time to debug

Reply:
The conditions aren't needed, can be unified.